packages/babel-preset-env/src/debug.js

```diff
@@ -43,50 +37,56 @@ export const logPlugin = (item, targetVersions, list, context) => {
     .replace(/^\{"/, '{ "')
     .replace(/"\}$/, '" }');
 
-  logMessage(`${item} ${formattedTargets}`, context);
+  console.log(`  ${item} ${formattedTargets}`);
 };
 
 export const logEntryPolyfills = (
+  polyfillName,
   importPolyfillIncluded,
   polyfills,
   filename,
-  onDebug,
+  polyfillTargets,
+  allBuiltInsList,
 ) => {
   if (!importPolyfillIncluded) {
-    console.log(
-      `
-[${filename}] \`import '@babel/polyfill'\` was not found.`,
-    );
+    console.log(`\n[${filename}] Import of ${polyfillName} was not found.`);
     return;
   }
   if (!polyfills.size) {
     console.log(
-      `
-[${filename}] Based on your targets, none were added.`,
+      `\n[${filename}] Based on your targets, polyfills were not added.`,
     );
     return;
   }
 
   console.log(
-    `
-[${filename}] Replaced \`@babel/polyfill\` with the following polyfill${wordEnds(
+    `\n[${filename}] Replaced ${polyfillName} entries with the following polyfill${wordEnds(
       polyfills.size,
     )}:`,
   );
-  onDebug(polyfills);
+  for (const polyfill of polyfills) {
+    logPluginOrPolyfill(polyfill, polyfillTargets, allBuiltInsList);
+  }
 };
 
-export const logUsagePolyfills = (polyfills, filename, onDebug) => {
+export const logUsagePolyfills = (
+  polyfills,
+  filename,
+  polyfillTargets,
+  allBuiltInsList,
+) => {
   if (!polyfills.size) {
     console.log(
-      `
-[${filename}] Based on your code and targets, none were added.`,
+      `\n[${filename}] Based on your code and targets, core-js polyfills were not added.`,
     );
     return;
   }
   console.log(
-    `
-[${filename}] Added following polyfill${wordEnds(polyfills.size)}:`,
+    `\n[${filename}] Added following core-js polyfill${wordEnds(
+      polyfills.size,
+    )}:`,
   );
-  onDebug(polyfills);
```

Comment:
This wasn't part of the public API, right?

Reply:
If internal `preset-env` plugins are not public API - not.